src/Symfony/Component/Translation/Bridge/Crowdin/CrowdinProviderFactory.php

```diff
@@ -66,16 +67,13 @@ public function create(Dsn $dsn): ProviderInterface
         $host = 'default' === $dsn->getHost() ? $this->getHost($dsn) : $dsn->getHost();
         $endpoint = sprintf('%s%s', $host, $dsn->getPort() ? ':'.$dsn->getPort() : '');
 
-        $filesDownloader = $this->client;
-
-        $client = $this->client->withOptions([
-            'base_uri' => 'https://'.$endpoint,
+        $client = ScopingHttpClient::forBaseUri($this->client, sprintf('https://%s/projects/%d/', $endpoint, $this->getUser($dsn)), [
             'headers' => [
                 'Authorization' => 'Bearer '.$this->getPassword($dsn),
             ],
```

Comment:
`['auth_bearer' => $this->getPassword($dsn)]` maybe?

Reply:
fixed thx